pyvrp/tests/test_read.py

```diff
@@ -1,29 +1,32 @@
-import ctypes
-
+import numpy as np
 from numpy.testing import assert_equal, assert_raises
 from pytest import mark
 
+from pyvrp.Matrix import IntMatrix  # noqa: F401
 from pyvrp.tests.helpers import read
 
 
 @mark.parametrize(
     "where, exception",
     [
-        ("data/UnknownEdgeWeightFmt.txt", RuntimeError),
-        ("data/UnknownEdgeWeightType.txt", RuntimeError),
-        ("somewhere that does not exist", ValueError),
-        ("data/FileWithUnknownSection.txt", RuntimeError),
-        ("data/DepotNotOne.txt", RuntimeError),
-        ("data/DepotSectionDoesNotEndInMinusOne.txt", RuntimeError),
-        ("data/MoreThanOneDepot.txt", RuntimeError),
-        ("data/NonZeroDepotServiceDuration.txt", RuntimeError),
-        ("data/NonZeroDepotReleaseTime.txt", RuntimeError),
-        ("data/NonZeroDepotOpenTimeWindow.txt", RuntimeError),
-        ("data/NonZeroDepotDemand.txt", RuntimeError),
-        ("data/TimeWindowOpenEqualToClose.txt", RuntimeError),
-        ("data/TimeWindowOpenLargerThanClose.txt", RuntimeError),
-        ("data/EdgeWeightsNoExplicit.txt", RuntimeError),
-        ("data/EdgeWeightsNotFullMatrix.txt", RuntimeError),
+        ("data/UnknownEdgeWeightFmt.txt", ValueError),
+        ("data/UnknownEdgeWeightType.txt", ValueError),
+        ("somewhere that does not exist", FileNotFoundError),
+        ("data/FileWithUnknownSection.txt", ValueError),
+        ("data/DepotNotOne.txt", ValueError),
+        # TODO: check must be done by VRPLIB
+        # ("data/DepotSectionDoesNotEndInMinusOne.txt", RuntimeError),
+        ("data/MoreThanOneDepot.txt", ValueError),
+        ("data/NonZeroDepotServiceDuration.txt", ValueError),
+        # TODO: release times not in VRBLIB format, so not supported by VRPLIB
+        # ("data/NonZeroDepotReleaseTime.txt", ValueError),
```

Comment:
Question: does `vrplib` not work with this file? It looks like it's correctly formatted in `VRPLIB` style.

Reply:
My bad :) fixed now